packaging/generate_package.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash -e
+
+VARIANT=$1
+if [[ ${VARIANT} == "deb" ]]; then
+    if [ -z "$2" ]; then
+        echo "Error, OS name missing for deb package type"
+        exit 1
+    else
+        OS=$2
+    fi
+fi
+
+VERSION_NO_SUFFIX="1.0.0"
+VERSION_SUFFIX="dev"
+VERSION="$VERSION_NO_SUFFIX"-"$VERSION_SUFFIX"
+
+SCRIPT=$(readlink -f "$0")
+DIR=$(dirname "$SCRIPT")
+export DIR
+
+ORIGINAL_DIR=`pwd`
+cd $DIR
+
+# Using CMAKE_BINARY_DIR uses an absolute path and will break cross-vm building/download/make functionality
+BUILD_DIR="$DIR/../"
+
+VENDOR="EOSNetworkFoundation"
+PROJECT_PREFIX="ANTELOPEIO"
+PROJECT="DUNE"
+DESC="Docker Utilities for Node Execution (DUNE) is a tool to abstract over Leap programs, CDT and other services/tools related to Antelope blockchains."
+URL="https://github.com/AntelopeIO/DUNE"
+EMAIL="support@eosnetwork.com"
+
+export BUILD_DIR
+export VERSION_NO_SUFFIX
+export VERSION_SUFFIX
+export VERSION
+export VENDOR
+export PROJECT_PREFIX
+export PROJECT
+export DESC
+export URL
+export EMAIL
+
+mkdir -p tmp
+
+if [[ ${VARIANT} == "brew" ]]; then
+   . "$DIR"./generate_bottle.sh
```

Comment:
after I fixed manually path, I've got this error:

./generate_package.sh brew
Error, unsupported OS X version

I have Monterey 12.6 was it intended?

Reply:
I copied the scripts from CDT. After checking it seems that supported macOS versions are quite old.
We need to update the scripts. I have created new issue for it: https://github.com/AntelopeIO/DUNE/issues/92